benchmarks/bench_linalg_decompositions.py

```diff
@@ -37,8 +37,5 @@ def time_kak_decomposition(target):
     cirq.kak_decomposition(target)
 
 
-time_kak_decomposition.params = [  # type: ignore
-    [np.eye(4), SWAP, SWAP * 1j, CZ, CNOT, SWAP.dot(CZ)]
-    + [cirq.testing.random_unitary(4) for _ in range(10)]
```

Comment:
I assume these random unitaries were producing misleading variance in the benchmark?